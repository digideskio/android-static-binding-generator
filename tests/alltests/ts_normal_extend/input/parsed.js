var MyClass = (function (_super) {
    __extends(MyClass, _super);
    function MyClass() {
        _super.apply(this, arguments);
    }
    MyClass.prototype.MyClassMethod1 = function () {
    };
    MyClass.prototype.MyClassMethod2 = function () {
    };
    return MyClass;
})(android.view.ViewGroup);