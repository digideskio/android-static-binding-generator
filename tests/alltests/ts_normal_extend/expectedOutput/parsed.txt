android.view.ViewGroup*_frnal_prepareExtend_l62_c37__MyClass*MyClassMethod1,MyClassMethod2*